mage/tests.go

```diff
@@ -0,0 +1,74 @@
+package mage
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gravitational/magnet"
+	"github.com/gravitational/trace"
+	"github.com/magefile/mage/mg"
+)
+
+type Test mg.Namespace
+
+func (Test) All() {
+	//mg.SerialDeps(Build.Go, Build.BuildContainer)
```

Comment:
Let's either add a todo item describing what needs to happen here or remove the commented code.